roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -56,7 +56,7 @@ oom_score = {{ containerd_oom_score }}
 {% endfor %}
 {% for addr in containerd_insecure_registries %}
         [plugins."io.containerd.grpc.v1.cri".registry.mirrors."{{ addr }}"]
-          endpoint = ["{{ ([ addr ] | flatten ) | join('","') }}"]
+          endpoint = ["http://{{ ([ addr ] | flatten ) | join('","') }}"]
```

Comment:
https registry setting here and work fine

{% for registry, addr in containerd_registries.items() %}
        [plugins."io.containerd.grpc.v1.cri".registry.mirrors."{{ registry }}"]
          endpoint = ["{{ ([ addr ] | flatten ) | join('","') }}"]
{% endfor %}

If endpoint is no http it will always try with https

Reply:
How about updating sample inventory instead by adding "http://"?
Current sample is
```
# containerd_insecure_registries:
#   - mirror.registry.io
#   - 172.19.16.11:5000

# containerd_registries:
#   "docker.io": "https://registry-1.docker.io"
```
As we see, the sample of containerd_registries contains "https://".
It would be consistent by adding "http://" to the sample of containerd_insecure_registries, I guess.

Reply:
thank you for your  advice,

i fixed code and doc for consistent